tests/unit/mocks/exceptions_mock.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2023, Denys Denysyev (ddenysyev@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING.GPLv3 or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import fusion as purefusion
+
+from http import HTTPStatus
+
+
+class HttpResponseMock:
+    def __init__(self, status, reason, body, headers=None):
+        self.status = status
+        self.reason = reason
+        self.body = body
+
+
+class ApiExceptionsMock:
```

Comment:
I'd rename it to `ApiMockExceptionsGenerator` or add some comment so it's visible that this class is for generating mock errors rather than for mock itself.

Reply:
@DenDen039 you gave the comment a 👍 but didn't change the code, maybe mistake?